app/scripts/snaps/preinstalled-snaps.ts

```diff
@@ -1,55 +1,14 @@
 import type { PreinstalledSnap } from '@metamask/snaps-controllers';
 
-// Preinstalled snaps requires fs and require to read and use any installed snaps.
-// We can switch `require` to `import`, but then this file gets transpiled to ESM & won't have access to `require.resolve`.
-// Work is needed to create the require object in ESM - specifically the import url. See snippet
-// import { createRequire } from 'node:module'
-// const require = createRequire(import.meta.url)
-//                                ^ The 'import.meta' meta-property is not allowed in files which will build into CommonJS output
-/* eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires */
-const fs = require('fs');
+// TODO - use the new version of the snap (once released), it should have this file.
+// Interim - build the snap locally (using `yarn build:clean`) and copy/paste this file & update the snap package.json exports to match.
+import MessageSigningSnap from '@metamask/message-signing-snap/dist/preinstalled-snap.json';
 
-const PREINSTALLED_SNAPS: PreinstalledSnap[] = [];
-
-///: BEGIN:ONLY_INCLUDE_IF(snaps)
```

Comment:
We don't actually need the code fence. See metamask controller, the import already has a code fence.